basic.py

```diff
@@ -90,3 +91,33 @@ def eval_(event):
         return eval(input, {"__builtins__": None}, safe_dict)
     except Exception as e:
         return "eval: {0}".format(e)
+
+
+last_events = {}
+
+
+@event
+def sub_event_catcher(event):
+    if event.event_type in ['message', 'cmd']:
+        last_events[event.meta['host']] = event
+
+
+@match(regex=r'^s/(.*?)/(.*?)/([a-z]*)$')
```

Comment:
@pepol like what? Any two separators? That doesn't seem to be matchable by a regular expression. We could in theory match just that initial `s` and try to parse anything afterwards but that kind of breaks the purpose of `@match`.